docs/index.rst

```diff
@@ -35,18 +46,43 @@ Then run the following command to start Kafka connect with the HDFS connector::
   $ ./bin/connect-standalone etc/schema-registry/connect-avro-standalone.properties \
   etc/kafka-connect-hdfs/quickstart-hdfs.properties
 
-You should see the process start up and log some messages, and then it will export data from
-Kafka to HDFS. In order to check that the data have been copied to HDFS, you can use Hive to query
-data or Avro tools to extract the file in HDFS::
+You should see that the process starts up and logs some messages, and then it will export data from
```

Comment:
Perhaps before asking them to run the connector, point them to the configuration file and ask them to validate important properties (such as the HDFS URI)?